src/storage/src/sst.rs

```diff
@@ -285,19 +295,71 @@ impl FileHandleInner {
     }
 }
 
+#[derive(Debug, Snafu, PartialEq)]
+pub struct ParseIdError {
+    source: uuid::Error,
+}
+
+/// Unique id for [SST File].
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct FileId(Uuid);
+
+impl FileId {
+    /// Returns a new unique [FileId] randomly.
+    pub fn random() -> FileId {
+        FileId(Uuid::new_v4())
+    }
+
+    /// Parses id from string.
+    pub fn parse_str(input: &str) -> std::result::Result<FileId, ParseIdError> {
+        Uuid::parse_str(input).map(FileId).context(ParseIdSnafu)
+    }
+
+    /// Append `.parquet` to file id to make a complete file name
+    pub fn as_parquet(&self) -> String {
+        format!("{}{}", self.0.hyphenated(), ".parquet")
+    }
+}
+
+impl fmt::Display for FileId {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+impl FromStr for FileId {
+    type Err = ParseIdError;
+
+    fn from_str(s: &str) -> std::result::Result<FileId, ParseIdError> {
+        FileId::parse_str(s)
+    }
+}
+
 /// Immutable metadata of a sst file.
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct FileMeta {
     /// Region of file.
     pub region_id: RegionId,
-    /// File name
-    pub file_name: String,
+    /// Compared to normal file names, FileId ignore the extension
+    #[serde(deserialize_with = "deserialize_from_string")]
+    #[serde(alias = "file_id")]
+    #[serde(alias = "file_name")]
+    pub file_id: FileId,
```

Comment:
> Deserialize this field from the given name or from its Rust name.

https://serde.rs/field-attrs.html#alias

According to the docs, we don't need to add an alias for `file_id`.
```suggestion
    #[serde(alias = "file_name")]
    pub file_id: FileId,
```

Reply:
Done! And I've also rebased my branch to `develop` to solve conflicts. It is ready to merge now.